bin/post-checkout.js

```diff
@@ -0,0 +1,24 @@
+// MIT © Sindre Sorhus - sindresorhus.com
+// via https://gist.github.com/sindresorhus/7996717
+
+const { execFile } = require('child_process');
+const [origHead, head, flag] = process.env.GIT_PARAMS.split(' ');
+
+// Flag is 1 if we moved between branches. Flag is 0 if we merely checked out a file from another branch.
+if (flag !== '1') {
+  return;
+}
+
+execFile(
+  'git',
+  ['diff-tree', '-r', '--name-only', '--no-commit-id', origHead, head],
+  (error, stdout, stderr) => {
+    if (error) {
+      console.error('stderr', stderr);
+      throw error;
+    }
+    if (stdout.includes('yarn.lock')) {
+      console.log('🎅 yarn.lock changed; RUN: yarn install');
+    }
+  }
+);
```

Comment:
I'm not so happy about repeating this over in all files. Would you like that I extract it in a separate file ?